### ISSUE TITLE
Accessibility: Images should have a text alternative or presentational role

### ISSUE THREAD

**Issue by dangerdak** [FAC-11/week3-TAAA#37]
The youtube thumbnail images should have alt text (even if it's empty) or `role="presentation"`

https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_text_02
